scapy/__init__.py

```diff
@@ -7,7 +7,7 @@
 Scapy: create, send, sniff, dissect and manipulate network packets.
 
 Usable either from an interactive console or as a Python library.
-http://www.secdev.org/projects/scapy
+https://scapy.net
```

Comment:
Is this change needed?

Reply:
I think it's better to point to the website rather than to the repository, so I'd say yes.